app/models/concerns/media_resources/soft_delete.rb

```diff
@@ -4,16 +4,18 @@ module SoftDelete
       extend ActiveSupport::Concern
 
       included do
-        scope :not_deleted, -> { where(deleted_at: nil) }
-        scope :deleted, -> { where.not(deleted_at: nil) }
+        scope :not_deleted, -> { where("#{table_name}.deleted_at IS NULL").or(where("#{table_name}.deleted_at > ?", Time.current)) }
+        scope :deleted, -> { where.not("#{table_name}.deleted_at IS NULL").where("#{table_name}.deleted_at <= ?", Time.current) }
 
         def soft_delete
           update!(deleted_at: Time.current)
         end
 
         def self.delete_soft_deleted
-          unscoped.where('deleted_at < ?', 6.month.ago).each do |resource|
+          # puts ">o> delete_soft_deleted"
+          unscoped.where("#{table_name}.deleted_at < ?", 6.months.ago).find_each do |resource|
             begin
+              puts ">o> Deleting soft deleted resource: #{resource.id}"
```

Comment:
i guess forgotten. otherwise better to use the proper log levels like `Rails.logger.debug` etc. because even if one forgets the prod log does not get unnecessarily cluttered.